_includes/charts/calico/templates/rbac.yaml

```diff
@@ -114,6 +114,7 @@ rules:
       - pods
       - nodes
       - namespaces
+      - configmaps
```

Comment:
I think we should put this in its own section - note the comment above this section is referring to the CNI plugin.

However, the CNI plugin doesn't need access to configmaps.

Reply:
Fixed, I think